conda-installation.md

```diff
@@ -33,18 +46,20 @@ and then use a tool called `conda pack` to package it up for running jobs.
 
 ## 1. Create a Miniconda installation
 
-Download the latest Linux [miniconda installer](https://docs.conda.io/en/latest/miniconda.html) and run it. 
+On the submit server, 
+download the latest Linux [miniconda installer](https://docs.conda.io/en/latest/miniconda.html) and run it. 
 
 ```
+[alice@submit]$ wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
```

Comment:
how stable is this link?

Reply:
Very. I don't think it's changed in years.